modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_dependency 'claims_api/application_controller'
+require_dependency 'claims_api/unsynchronized_evss_claims_service'
+
+module ClaimsApi
+  module V1
+    class ClaimsController < ApplicationController
+      before_action { permit_scopes %w[claim.read] }
+      before_action :verify_power_of_attorney
+
+      def index
+        claims = service.all
+        render json: claims,
+               serializer: ActiveModel::Serializer::CollectionSerializer,
+               each_serializer: ClaimsApi::ClaimListSerializer
+      end
+
+      def show
+        if (pending_claim = ClaimsApi::AutoEstablishedClaim.pending?(params[:id]))
+          render json: pending_claim,
+                 serializer: ClaimsApi::AutoEstablishedClaimSerializer
+        else
+          evss_claim_id = ClaimsApi::AutoEstablishedClaim.evss_id_by_token(params[:id]) || params[:id]
+          fetch_and_render_evss_claim(evss_claim_id)
+        end
+      end
+
+      private
+
+      def fetch_and_render_evss_claim(id)
+        claim = service.update_from_remote(id)
+        render json: claim, serializer: ClaimsApi::ClaimDetailSerializer
+      end
+
+      def service
+        ClaimsApi::UnsynchronizedEVSSClaimService.new(target_veteran)
+      end
+
+      def target_veteran
+        ClaimsApi::Veteran.from_headers(request.headers)
+      end
+
+      def verify_power_of_attorney
```

Comment:
we should remove all PoA related stuff from V0 and just leave it in V1, not sure if you want to do it here or have @charleystran do it on his PR, I'll let you two decide who wants to do it.

Reply:
resolved in https://github.com/department-of-veterans-affairs/vets-api/pull/2927/commits/9f7d6f6cfefe879a4ece50b3b6c9f805ee8a0cb7